exotica/Vector q0
geometry_msgs/Pose goal_pose
moveit_msgs/PlanningScene ps
int32[] invalid_samples
---
bool succeed
exotica/Vector q_out
int32 sample_index
---
int32 goal_cell_index

